polaris-for-figma/src/utils.ts

```diff
@@ -48,12 +47,12 @@ const backgroundTokens: Token[] = tokensArray.filter(({figmaName}) =>
 const surfaceTokens: Token[] = tokensArray.filter(({figmaName}) =>
   figmaName.startsWith('Surface'),
 );
-const shadowTokens: Token[] = tokensArray.filter(({figmaName}) =>
-  figmaName.startsWith('Shadow'),
-);
-const hintTokens: Token[] = tokensArray.filter(({figmaName}) =>
-  figmaName.startsWith('Hint'),
-);
+// const shadowTokens: Token[] = tokensArray.filter(({figmaName}) =>
```

Comment:
We're about to add this functionality to the plugin in https://github.com/Shopify/polaris/issues/5398 so commenting out instead of deleting